.gitignore

```diff
@@ -53,6 +53,8 @@ Base.rte/Settings.ini
 Benchmark.rte/
 Scenes.rte/
 Metagames.rte/
+Saves.rte/*
+!Saves.rte/Index.ini
```

Comment:
I guess this is more explicit about it, but you can also just follow the same gitignore structure as, say Metagames.rte. Ignore the folder but force add the index ini and it won't be ignored.
Change it if you want to, or don't, I just wanted to mention it.